src/main/java/com/synopsys/integration/alert/provider/blackduck/actions/BlackDuckGlobalTestAction.java

```diff
@@ -63,7 +64,13 @@ public MessageResult testConfig(String configId, String description, FieldAccess
         String apiToken = fieldAccessor.getStringOrEmpty(BlackDuckDescriptor.KEY_BLACKDUCK_API_KEY);
         String url = fieldAccessor.getStringOrEmpty(BlackDuckDescriptor.KEY_BLACKDUCK_URL);
         String timeout = fieldAccessor.getStringOrEmpty(BlackDuckDescriptor.KEY_BLACKDUCK_TIMEOUT);
-        BlackDuckProperties blackDuckProperties = blackDuckPropertiesFactory.createProperties(Long.valueOf(configId), fieldAccessor);
+        Long parsedConfigurationId = ProviderProperties.UNKNOWN_CONFIG_ID;
+
+        if (StringUtils.isNotBlank(configId)) {
```

Comment:
Should we validate if configId is a number?

Reply:
If a configId isn't a number then something is very wrong. I feel it should break in that case.

Reply:
If someone is using REST calls, configId may not be a number.